amm/contracts/router/lib.rs

```diff
@@ -7,31 +7,40 @@ pub mod router {
     const TRADING_FEE_ADJ_DENOM: u128 = 1000;
     const TRADING_FEE_ADJ_NUM: u128 = 997;
```

Comment:
Why is this not affected by this PR? This 997/1000 stands for the "1 minus fee" multiplier used in all the "get_amountX" calculations. Since now fee can be either 0.003 or 0.001, this should probably be taken into account here in router

Reply:
Ah, you're right.